src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1109,73 +1101,67 @@ RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
   }
 
 #ifdef ENABLE_SECURITY
-  // for packet is encrypted and security plugin is enable
-  if (protection_requested && security_manager_) {
+  // for packet is encrypted and security plug-in is enable
```

Comment:
@tkireva This comment could be a bit misleading, please update it or just remove it